MovieApp/Infrastructure/Views/MovieDetails/MovieDetailsViewController.swift

```diff
@@ -0,0 +1,358 @@
+import UIKit
+import PureLayout
+
+class MovieDetailsViewController: UIViewController {
+    
+    private var viewModel: MovieDetailsViewModel?
+    var movieId: Int?
+    
+    let RatingLabel = UILabel()
+    let Userscore = UILabel()
+    let MovieNameLabel = UILabel()
+    let YearLabel = UILabel()
+    let DateLabel = UILabel()
+    let MovieDurationLabel = UILabel()
+    let CategoryLabel = UILabel()
+    let OverviewLabel = UILabel()
+    let MovieSummaryLabel = UILabel()
+    
+    let ScreenView1 = UIView()
+    let ScreenView2 = UIView()
+    
+    let stackView = UIStackView()
+    
+    let movieImageView = UIImageView()
+    
+    var allMembers: [String] = []
+    var allRoles: [String] = []
+    
+    let MemberLabel1 = UILabel()
+    let MemberLabel2 = UILabel()
+    let MemberLabel3 = UILabel()
+    let MemberLabel4 = UILabel()
+    let MemberLabel5 = UILabel()
+    let MemberLabel6 = UILabel()
+    let Role1 = UILabel()
+    let Role2 = UILabel()
+    let Role3 = UILabel()
+    let Role4 = UILabel()
+    let Role5 = UILabel()
+    let Role6 = UILabel()
```

Comment:
We always use camelCase for variables. That means `MemberLabel3` should actually be `memberLabel3`